marked-element.html

```diff
@@ -149,17 +158,35 @@
         observer: 'render',
         type: Function,
         value: null
+      },
+      /**
+       * A reference to the XMLHttpRequest instance used to generate the
+       * network request.
+       *
+       * @type {XMLHttpRequest}
+       */
+      xhr: {
+        type: Object,
+        notify: true,
+        readOnly: true,
+        value: function() {
+          return new XMLHttpRequest();
+        }
       }
     },
 
     ready: function() {
-      if (!this.markdown) {
-        // Use the Markdown from the first `<script>` descendant whose MIME type starts with
-        // "text/markdown". Script elements beyond the first are ignored.
-        var markdownElement = Polymer.dom(this).querySelector('[type^="text/markdown"]');
-        if (markdownElement != null) {
-          this.markdown = this._unindent(markdownElement.textContent);
-        }
+      if (this.markdown) return;
+
+      // Use the Markdown from the first `<script>` descendant whose MIME type starts with
+      // "text/markdown". Script elements beyond the first are ignored.
+      var markdownElement = Polymer.dom(this).querySelector('[type^="text/markdown"]');
```

Comment:
It looks like `text/markdown` is the only thing registered for markdown in the IANA and RFC7763. Would you prefer to use `[type="text/markdown"]` here instead?